scanomatic/ui_server_data/js/ccc/containers/ImageUploadContainer.jsx

```diff
@@ -35,13 +35,13 @@ export default class ImageUploadContainer extends React.Component {
 
   handleUploadError(reason) {
     this.setState({ image: null, progress: null });
+    // eslint-disable-next-line no-alert
     alert(`An error occured while uploading the image: ${reason}`);
   }
 
   render() {
     return (
       <ImageUpload
-        image={this.state.image}
```

Comment:
This was never used in `<ImageUpload/>`, but there are a couple of tests that check how this is propagated from `<ImagUploadContainer/>`. Not sure what to do about those...

Reply:
Here are the tests:
https://github.com/Scan-o-Matic/scanomatic-standalone/blob/bdeb43fc3196f446ebf6461dfc40650c73c2ed13/scanomatic/ui_server_data/js/specs/containers/ImageUploadContainer.spec.jsx#L32-L44

Reply:
Removed tests in https://github.com/Scan-o-Matic/scanomatic-standalone/pull/64/commits/9d788e6f40b3391985cab692aff62b9ef24c8950 since I suspect they test something which was never needed.